packages/auth/src/providers/cognito/apis/confirmResetPassword.ts

```diff
@@ -43,17 +47,24 @@ export async function confirmResetPassword(
 	);
 	const metadata = input.options?.serviceOptions?.clientMetadata;
 
+	const UserContextData = getUserContextData({
+		username,
+		userPoolId,
+		userPoolClientId,
+	});
+
 	await confirmForgotPassword(
-		{ 
-			region: getRegion(authConfig.userPoolId), 
-			userAgentValue: getAuthUserAgentValue(AuthAction.ConfirmResetPassword)
+		{
+			region: getRegion(authConfig.userPoolId),
+			userAgentValue: getAuthUserAgentValue(AuthAction.ConfirmResetPassword),
 		},
 		{
 			Username: username,
 			ConfirmationCode: code,
 			Password: newPassword,
 			ClientMetadata: metadata,
 			ClientId: authConfig.userPoolClientId,
+			UserContextData: UserContextData,
```

Comment:
I believe this is a ts lint rule

```suggestion
                        UserContextData
```